index.js

```diff
@@ -1587,6 +1597,32 @@ ContainerWindow.prototype = {
 
     return true;
   },
+
+  _shutdownContainers() {
+    ContextualIdentityProxy.getIdentities().forEach(identity => {
```

Comment:
Can we add a comment explaining this whole function.

`// Changes icons back to defaults when they are not in native firefox`

or something similar